src/pages/index.vue

```diff
@@ -70,14 +70,32 @@
 </template>
 
 <script>
-import { definePageMeta } from '#imports';
+import {
+  definePageMeta,
+  useSeoMeta,
+} from '#imports';
 import GenericPanel from '@/components/commons/GenericPanel.vue';
+import Constants from '~/constants';
```

Comment:
Use destructing. Also don't use Pascal case to JS files. It should only apply to component imports. Sorry this is not clear, but I will make a guide about this. Thanks

Reply:
Applied changes using meta.js